lib/game.rb

```diff
@@ -109,24 +130,97 @@ def player_sub_placement
     @player_board.place(@player_sub, @player_sub_coords)
   end
 
-  def turn_phase
+  def round_start
     puts "=============COMPUTER BOARD============="
     @board.render
     puts "==============PLAYER BOARD=============="
     @player_board.render(true)
+  end
 
+  def player_shot
     loop do
       puts "Enter the coordinate for your shot"
       player_shot = gets.chomp
-      formatted_player_shot = player_shot.to_s
+      @formatted_player_shot = player_shot.to_s
 
-      if !@board.valid_coordinate?(formatted_player_shot) || @board.cells[formatted_player_shot].fired_upon? 
+      if !@board.valid_coordinate?(@formatted_player_shot) || @board.cells[@formatted_player_shot].fired_upon? 
         puts "Please enter a valid coordinate"
       end
 
-      break if @board.valid_coordinate?(formatted_player_shot) && !@board.cells[formatted_player_shot].fired_upon?
+      break if @board.valid_coordinate?(@formatted_player_shot) && !@board.cells[@formatted_player_shot].fired_upon?
+
     end #Not receiving any errors, but we will need to test the "re-shoot on a cell logic"
+
+    @board.cells[@formatted_player_shot].fire_upon
+    @board.cells[@formatted_player_shot].render
+  end
+
+  def computer_shot
+
+    loop do
+      @c_shot = @board.cells.keys.sample
+
+      break if !@board.cells[@c_shot].fired_upon? 
+      # && !@computer_shots.include?(@c_shot)
+    end
+
+    #Although our fired upon? should work, I kept on getting repeats.  I added the @compuer_shots for redundancy until we can figure out the problem
```

Comment:
repeats on the same cell? wondering if we want to add a conditional for if the new input matches the last?

I'll look into ways we might be able to do that